src/api/general-services/backend-status.js

```diff
@@ -3,14 +3,14 @@ const getPipelineStatus = require('./pipeline-status');
 const getWorkerStatus = require('./worker-status');
 
 const getBackendStatus = async (experimentId) => {
-  const [{ pipeline }, { gem2s }, { worker }] = await Promise.all(
+  const [{ qc }, { gem2s }, { worker }] = await Promise.all(
     [
       getPipelineStatus(experimentId, constants.QC_PROCESS_NAME),
       getPipelineStatus(experimentId, constants.GEM2S_PROCESS_NAME),
       getWorkerStatus(experimentId)],
   );
   return {
-    pipeline,
+    pipeline: qc,
```

Comment:
Renamed from `qc` to `pipeline` because it is still handled as `pipeline` in the UI

Reply:
afaik this is now fixed on Martin's UI pr (ui/277)